docs/advanced/signal_generators.md

```diff
@@ -0,0 +1,576 @@
+# Signal Generation
+
+An overview of the signal generation functionality available in this package, along with information about Tektronix signal generators.
+
+______________________________________________________________________
+
+## Function Generation
+
+A function is a limited set of common waveforms that are provided by default through the instrument.
+The simplicity of these waveforms allows for output parameters like waveform length and sample rate to be
+abstracted away. Frequency replaces these in order to provide signals which are easy to quantify and manipulate.
+
+Arbitrary Function Generators ({term}`AFG`s) utilize a phase increment process and a data lookup table to provide variable frequencies.
```

Comment:
Everywhere a term is used but the wording is modified needs to be updated in this way:
```suggestion
Arbitrary Function Generators ({term}`AFGs <AFG>`) utilize a phase increment process and a data lookup table to provide variable frequencies.
```